service/controller/v19/resource/cloudformation/main_stack_test.go

```diff
@@ -99,8 +99,9 @@ func TestMainGuestTemplateExistingFields(t *testing.T) {
 						IdleTimeoutSeconds: 3600,
 					},
 				},
-				Region: "eu-central-1",
-				AZ:     "eu-central-1a",
+				Region:            "eu-central-1",
+				AZ:                "eu-central-1a",
+				AvailabilityZones: 1,
```

Comment:
This must be fixed once the subnet PR is merged (as it will introduce two AZs for this test).